arch/x86/core/intel64/cpu.c

```diff
@@ -28,6 +28,11 @@ extern uint8_t _exception_stack1[];
 extern uint8_t _exception_stack2[];
 extern uint8_t _exception_stack3[];
 
+extern uint8_t _exception_stack4[];
```

Comment:
please don't call these 4,5,6,7 this is confusing, call them z_x86_nmi_stack or something like that

Reply:
renamed